multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/beans/SortBy.java

```diff
@@ -0,0 +1,47 @@
+
+package io.apicurio.multitenant.api.beans;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum SortBy {
+
+    name("name"),
+    createdOn("createdOn"),
+    tenantId("tenantId");
```

Comment:
Use uppercase?

Reply:
I think it's better like this because it's equal to the fields in the RegistryTenant object